multinet/auth/__init__.py

```diff
@@ -0,0 +1 @@
+"""Authorization types."""
```

Comment:
Looks like it's authorization according to the OAuth [website](https://oauth.net/). This makes sense, because what's really going on is that the account holder (whoever's logging in) is giving the requesting app (multinet) authorization to access their account info.

Reply:
:+1: